src/runtime/stores/formatters.ts

```diff
@@ -57,6 +57,14 @@ const formatMessage: MessageFormatter = (id, options = {}) => {
     }
 
     message = defaultValue ?? id;
+
+    if (getOptions().onMissingMessageHandler) {
+      getOptions().onMissingMessageHandler(
```

Comment:
Semi-related question: would it make sense to be able to return the message string from the missing message handler? Something like:

![CleanShot 2022-04-05 at 14 18 59@2x](https://user-images.githubusercontent.com/12702016/161752061-a9efdfc8-e4da-41fd-b15b-f6a775962219.jpg)

Reply:
not for my case, but maybe we should add that...
Do you want me to adapt it?

Reply:
just made a new commit...

Reply:
did also the rename from `onMissingMessageHandler` to `handleMissingKey`